internal/pkg/term/selector/local_selector.go

```diff
@@ -145,6 +145,61 @@ func NewLocalFileSelector(prompt Prompter, fs afero.Fs) (*localFileSelector, err
 	}, nil
 }
 
+// StaticSources asks the user to select from a list of directories and files in the current directory and two levels down.
```

Comment:
💭 i wonder if we should prompt for directories based on the workspace dir? (git root)? just bc that's how it goes into the manifest file? i'm not sure which is better tbh

Reply:
If they input a custom path, should we require it to be based on the workspace dir, too, or accept either and conditionally translate it after the fact? 🤔

Reply:
I feel like maybe it's ok to limit the input dir to be ws only for now? We could have some logic in manifest validation too and then adjust ad hoc.

Reply:
What do you all think of this?
```
 % copilot svc init -t "Static Site" -n djf
Note: It's best to run this command in the root of your workspace.

  Which files or directories would you like to upload for djf?
Directories are recursively uploaded.  [Use arrows to move, space to select, type to filter, ? for more help]
  [ ]  /Users/jwh/ECS-DevX/sample-apps/aws-copilot-sample-service/appspec.yaml
  [ ]  /Users/jwh/ECS-DevX/sample-apps/aws-copilot-sample-service/create-deployment.json
  [ ]  /Users/jwh/ECS-DevX/sample-apps/aws-copilot-sample-service/demo-app
  > [ ]  /Users/jwh/ECS-DevX/sample-apps/aws-copilot-sample-service/demo-app/thisfile.go
  [ ]  /Users/jwh/ECS-DevX/sample-apps/aws-copilot-sample-service/demo-app/thisfolder
  [ ]  /Users/jwh/ECS-DevX/sample-apps/aws-copilot-sample-service/demo-app/thisfolder/anotherfile.go
  [ ]  /Users/jwh/ECS-DevX/sample-apps/aws-copilot-sample-service/index.html
```
I feel like it makes more sense to the user because they don't think in terms of the workspace dir. Then we'll translate it to be relative to the workspace dir before populating it in the manifest.

Reply:
> they don't think in terms of the workspace dir.

This is true, but I feel like they do think in terms of a project root. For example, if their folder structure is like:
```
+ aws-copilot-sample-service/
    +- copilot/
          - .workspace
    +- demo-app/
          +- thisfolder
                - anotherfile.go
    - appsec.yml
    - index.html
```
Then I think it would probably make sense to prompt for 
```
[ ] demo-app
[ ] demo-app/thisfolder
[ ] demo-app/thisfolder/anotherfile.go
[ ] appsec.yml
[ ] index.html
```

Because `aws-copilot-sample-service/` is the project root?

Reply:
Can we assume the project root is the `cwd`?

Reply:
I think today users are able to run commands that require a workspace, such as `copilot svc deploy` and `copilot svc init`, *not* from the project root, but from dirs like `aws-copilot-sample-service/copilot` or `aws-copilot-sample-service/demo-app`, which are within the project, but not necessarily the root.

Therefore, I think it'd make sense for `copilot svc init` with static site to make the same level of assumptions. What do yall think 🤔 ?

Reply:
That makes sense to me. 👍🏼